editor/editor_interface.h

```diff
@@ -69,11 +70,13 @@ class EditorInterface : public Object {
 	PropertySelector *property_selector = nullptr;
 	PropertySelector *method_selector = nullptr;
 	SceneTreeDialog *node_selector = nullptr;
+	CreateDialog *create_dialog = nullptr;
 
 	void _node_selected(const NodePath &p_node_paths, const Callable &p_callback);
 	void _property_selected(const String &p_property_name, const Callable &p_callback);
 	void _method_selected(const String &p_property_name, const Callable &p_callback);
 	void _quick_open(const String &p_file_path, const Callable &p_callback);
+	void _create_dialog_item_selected(const bool p_is_cancelled, const Callable &p_callback);
```

Comment:
```suggestion
	void _create_dialog_item_selected(bool p_is_cancelled, const Callable &p_callback);
```
Don't use `const` with primitive arguments.